src/main/java/com/strandls/cca/service/impl/CCADataServiceImpl.java

```diff
@@ -688,4 +708,92 @@ public Boolean sendPermissionGrant(HttpServletRequest request, EncryptedKey encr
 		return false;
 	}
 
+
+	@Override
+	public Map<String, Object> searchCCAData(String query, HttpServletRequest request, UriInfo uriInfo)
```

Comment:
is it possible to have a POJO instead of a Map as return type?

Reply:
Possible to use SubsetCCADataList pojo, but have to modify ui codebase a bit.Should I do that?